.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+on: [push, pull_request]
+name: pREST CI
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+          
+        ports:
+        - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
when you create a Postgres with GitHub service, and inside the script `./testdata/schema.sh` we used psql without password you need to add on env  the           `PGPASSWORD: postgres
`